pkg/cmd/migrate.go

```diff
@@ -0,0 +1,74 @@
+package cmd
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"strconv"
+
+	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/pressly/goose/v3"
+	"github.com/spf13/cobra"
+)
+
+const (
+	migrationsDir           = "storage/postgres/migrations"
```

Comment:
Can we change this up so that there is a top-level migrations directory with subdirectories per engine? When we start adding other storage engines this will be harder to manage, because we'll have to include each directory in a COPY command to build our containers, instead of just copying the top-level migrations directory (and it's subdirectories).

Reply:
Yep. Thanks. Done.